packages/experiments/src/components/Button/Button.tsx

```diff
@@ -1,10 +1,13 @@
 import { createComponent } from '../../Foundation';
 import { useButtonState as state } from './Button.state';
-import { ButtonStyles as styles, ButtonTokens as tokens } from './Button.styles';
+import { ButtonClassNames, ButtonStyles as styles, ButtonTokens as tokens } from './Button.styles';
 import { IButtonProps } from './Button.types';
 import { ButtonView } from './Button.view';
 
+const classNames = Object.values(ButtonClassNames);
```

Comment:
Don't follow what's going on here; classNames would be an array of the values... what is that used for?

Reply:
We just need the values to make the mapping from `className` to `component`.

This is done in lines 95-101 of `createComponent.tsx`:

```ts
// If className list is present, add each className to dictionary.
if (options.classNames) {
   for (const className of options.classNames) {
      classNameToComponentDictionary[className] = result.displayName;
   }
}
```